### PR TITLE
Clothing grammar fixes, mostly

### PR DESCRIPTION
**Pull request by Churro55**
Went through most clothes and adjusted some grammar errors, capitalized faction names and ranks.
Lowered the triad jacket's armor value (approved by possum) because whoever made it gave it stats better than most armor vests. Lowered other coat's values because they didn't make sense and people powergamed.
Lowered some hunting lodge armor values because someone kept power creeping them to be better than their counterparts.

Let me know if you find any more grammar errors so I can change them.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -220,7 +220,11 @@
 
 /obj/item/clothing/suit/armor/vest/custodian
 	name = "custodian armor"
+<<<<<<< Updated upstream
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/armor.dm:223]: ?

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -331,10 +331,17 @@ obj/item/clothing/suit/fluff/kimono
 	icon_state = "triadkillers"
 	item_state = "triadkillers"
 	armor = list(
+<<<<<<< Updated upstream
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/miscellaneous.dm:334]: ???

File: code/modules/clothing/suits/armor.dm (hunk 1 above)
Comment (Churro55) [reply code/modules/clothing/suits/armor.dm]: Fixed, I think

File: code/modules/clothing/suits/miscellaneous.dm (hunk 2 above)
Comment (Churro55) [reply code/modules/clothing/suits/miscellaneous.dm]: Fixed too, I think

**Comment by Kazkin** [comment]
I am doubtful this compiles given the <<<<updated upstream unless that is a git thing.

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -220,7 +220,8 @@
 
 /obj/item/clothing/suit/armor/vest/custodian
 	name = "custodian armor"
-	desc = "Someone's gotta clean this mess. While this armor is technically church property, Lonestar custodians often use it on loan to protect themselves from the colonies' various dangerous pests."
+	desc = "Someone's gotta clean this mess. While this armor is technically church property, Lonestar custodians often use it on loan to protect themselves from the colonies various dangerous pests."
+>>>>>>> Stashed changes
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:224]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 4]
```diff
@@ -252,7 +253,11 @@
 
 /obj/item/clothing/suit/armor/vest/prime
 	name = "prime armor"
+<<<<<<< Updated upstream
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:256]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 5]
```diff
@@ -252,7 +253,11 @@
...
 	desc = "Trust in god, but keep your armor on."
+=======
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:258]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 6]
```diff
@@ -252,7 +253,11 @@
...
+	desc = "Trust in God but keep your armor on."
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:259]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 7]
```diff
@@ -252,7 +253,11 @@
...
+>>>>>>> Stashed changes
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:260]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 8]
```diff
@@ -280,7 +285,11 @@
 
 /obj/item/clothing/suit/armor/vest/soteriasuit
 	name = "'Mark II' environmental protection suit"
+<<<<<<< Updated upstream
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:288]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 9]
```diff
@@ -280,7 +285,11 @@
...
 	desc = "For working in hazardous environments. While its built for most environments, one of those is not space. This suit is a cheap and badly made copy of the Artificer Guilds original design. \
+=======
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:290]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 10]
```diff
@@ -280,7 +285,11 @@
...
+	desc = "For working in hazardous environments. While its built for most environments, one of those is not space. This suit is a cheap and badly made copy of the Artificer Guild's original design. \
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:291]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 11]
```diff
@@ -280,7 +285,11 @@
...
+>>>>>>> Stashed changes
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:292]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 12]
```diff
@@ -821,9 +830,15 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
 	icon_state = "hm_woodvest"
 	item_state = "hm_woodvest"
 	armor = list(
+<<<<<<< Updated upstream
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:833]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 13]
```diff
@@ -821,9 +830,15 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
...
 		melee = 25, //It's made of mostly wood and cloth, shittiest armor in the game easily, but does have bio and rad, giving it a rare but still possible boost over handmade. -Kaz
 		bullet = 15, // Justifying keeping somewhat decent values on bullet just because I changed the recipe to include steel. - Seb
 		energy = 10, // It's mostly made of wood, it will char easily.
+=======
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:837]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 14]
```diff
@@ -821,9 +830,15 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
...
+		melee = 30,
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:838]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 15]
```diff
@@ -821,9 +830,15 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
...
+		bullet = 30,
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:839]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 16]
```diff
@@ -821,9 +830,15 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
...
+		energy = 25,
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:840]:
```suggestion
```

File: code/modules/clothing/suits/armor.dm [hunk 17]
```diff
@@ -821,9 +830,15 @@ obj/item/clothing/suit/armor/commander/marshal_coat_ss
...
+>>>>>>> Stashed changes
```
Comment (r4d6) [inline code/modules/clothing/suits/armor.dm:841]:
```suggestion
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 18]
```diff
@@ -331,10 +331,11 @@ obj/item/clothing/suit/fluff/kimono
...
 	armor = list(
-		melee = 15, //Loadout item with pockets, cool looking and decent protection but easily outclassed. -Kaz
-		bullet = 15,
-		energy = 15,
-		bomb = 15,
+		melee = 10,
+		bullet = 5,
+		energy = 5,
+		bomb = 0,
+>>>>>>> Stashed changes
```
Comment (r4d6) [inline code/modules/clothing/suits/miscellaneous.dm:338]:
```suggestion
```

**Review by r4d6 — CHANGES_REQUESTED** [review]
This isn't how you update your fork and fix conflicts.

**Comment by Churro55** [comment]
I have absolutely no idea how those conflicts happened, they only appeared once submitted to github. 
Should be fixed now, thanks, @Trilbyspaceclone @Kazkin @r4d6

**Pull request merged**